import-export-cli/integration/testutils/testTypes.go

```diff
@@ -64,6 +64,16 @@ type ApiProductImportExportTestArgs struct {
 	IsLatest             bool
 }
 
+type ThrottlePolicyImportExportTestArgs struct {
+	Admin          Credentials
```

Comment:
I don't think we need Admin field here. Normally, these types of structs have fields corresponding to the respective apictl command(s) only (except for some exceptional cases).
Can we remove it?